server/routes/prisons/configuration/prisonConfigController.ts

```diff
@@ -2,17 +2,57 @@ import { RequestHandler } from 'express'
 import { PrisonService } from '../../../services'
 import { responseErrorToFlashMessage } from '../../../utils/utils'
 
-export default class PrisonStatusController {
+export default class PrisonConfigController {
   public constructor(private readonly prisonService: PrisonService) {}
 
   public view(): RequestHandler {
     return async (req, res) => {
       const { prisonId } = req.params
       const prison = await this.prisonService.getPrison(res.locals.user.username, prisonId)
 
-      return res.render('pages/prisons/status/status', {
+      // will need to add prisonId
+      const information = await this.prisonService.getPrisonConfig(res.locals.user.username)
+      if (information.email === '') {
+        information.email = 'Not set'
+      }
+      if (information.phone === '') {
+        information.phone = 'Not set'
+      }
+      if (information.website === '') {
+        information.website = 'Not set'
+      }
```

Comment:
not happy with this section - but it makes life simpler if doing this in the route level - as attaching data tests when you've got if statements on a value becomes problematic